starters/default/gatsby-config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
     author: `@gatsbyjs`,
     keywords: [`gatsby`, `application`, `react`],
-    image: `https://www.dropbox.com/s/bhwnfkuwewq03sl/gatsby-share.png?raw=1`,
+    image: `https://raw.githubusercontent.com/gatsbyjs/gatsby/master/starters/default/src/images/gatsby-share.png`,
```

Comment:
the image is checked into the repo and should be in the local folder, then a relative path would be enough

Reply:
@muescha Yes, but Open Graph doesn't support relative URLs - it's `content` not `src`, and only takes a "string of Unicode characters forming a valid URL having the http or https scheme"

Reference: https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url

Reply:
Let me know if it's alright to merge. Or if you have an alternative suggestion that would work.

Reply:
but this looks not right. it has every time to download a file from github which already exists in the folder. someone change later the pic in the folder and not understand why it not changed in the app

maybe someone other can help with this case?

note:
on the blog starter the fixed images are places in folder `assets` 
and in the `bio.js` the pic is referenced in a query - so it should be possible to reference the local image